libs/ml/include/ml/optimisation/optimiser.hpp

```diff
@@ -223,7 +250,7 @@ typename T::Type Optimiser<T, C>::Run(
   UpdateLearningRate();
   epoch_++;
 
-  return loss_sum / static_cast<DataType>(step);
+  return loss_sum;
```

Comment:
loss normalisation removed?

Reply:
Criterion which get batch will do averaging. It will be changed in next PR, because criterion will be op and will return Tensor of losses.